app/views/users/sessions/_passwd_small.html.erb

```diff
@@ -1,32 +1,29 @@
-<% # We need to override the ids to avoid accessibility issues on the new user
-   # session page, which has a second user session form %>
+<%# We need to override the ids to avoid accessibility issues on the new user session page,
+which has a second user session form %>
 <%= form_for(User.new, url: user_session_path, html: { id: "new_user_session_small" }) do |f| %>
-	<dl>
-    <dt>
-      <%= f.label :login, ts("User name or email:"),
-              for: "user_session_login_small" %></dt>
-    <dd><%= f.text_field :login,
-              id: "user_session_login_small" %></dd>
-    <dt><%= f.label :password, ts("Password:"),
-              for: "user_session_password_small" %></dt>
-    <dd><%= f.password_field :password,
-              id: "user_session_password_small" %></dd>
+  <dl>
+    <dt><%= f.label :login, t(".username_or_email"), for: "user_session_login_small" %></dt>
+    <dd><%= f.text_field :login, autocomplete: "on", id: "user_session_login_small" %></dd>
```

Comment:
It probably depends on browser, but I think the default is usually for autocomplete to be on. erb-lint prefers it to be explicit though (and especially for things like email, we can make the UX better by telling the browser it is an email and not freeform text)

Reply:
This is probably okay for log in, but for anywhere we save data (especially if we display the saved data publicly), we probably want it off to avoid people's wallet name/work email/etc getting filled, like in https://otwarchive.atlassian.net/issues/AO3-6000